### PR TITLE
TB Management - Progress report

### PR DESCRIPTION
**Pull request by anvmn**
#1206

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2024-06-27 10-57

File: client/src/elm/Pages/Tuberculosis/ProgressReport/Model.elm [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import Components.ReportToWhatsAppDialog.Model
+import EverySet exposing (EverySet)
+import Pages.Page exposing (Page)
+
+
+type alias Model =
+    { showEndEncounterDialog : Bool
+    , reportToWhatsAppDialog : Components.ReportToWhatsAppDialog.Model.Model
+
+    -- , components : Maybe (EverySet Components.ReportToWhatsAppDialog.Model.ReportComponentTuberculosis)
```
Comment (AronNovak) [inline client/src/elm/Pages/Tuberculosis/ProgressReport/Model.elm]: Will it be done in this PR or in follow-ups?

**Review by AronNovak — APPROVED** [review]
Looks good, basically the view part is missing, but it's `do not merge` yet.

**Comment by anvmn** [comment]
@AronNovak 

>  basically the view part is missing, but it's `do not merge` yet.

It's at #1226 and #1228

File: client/src/elm/Pages/Tuberculosis/ProgressReport/Model.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/Tuberculosis/ProgressReport/Model.elm]: Waiting for @adamhstewart to update on https://github.com/TIP-Global-Health/eheza-app/issues/1207#issuecomment-2194351795

**Pull request merged**